### PR TITLE
Erinhales/fix dark theme

### LINKED ISSUE #174: Dark Theme UI
Stats bar doesn't take on dark-theme UI changes:
Screen Shot 2021-01-07 at 2 37 32 PM

### Environment

* Elixir & Erlang/OTP versions (`elixir -version`):
* Operating System:
* Blockscout Version/branch: 

### Steps to reproduce

Click on the moon in the nav bar to enter dark mode

### PR DESCRIPTION
**Pull request by ErinHales**
Closes #174 

## Motivation

The banner wasn't taking on the dark theme styling

### Enhancements
Updated styling, confirmed it doesn't affect regular light theme styles.

Screen Shot 2021-01-21 at 2 20 37 PM
Screen Shot 2021-01-21 at 2 20 46 PM

Screen Shot 2021-03-02 at 11 09 41 AM

### CODE REVIEW

File: apps/block_scout_web/assets/css/theme/_dai_variables.scss [hunk 1]
```diff
@@ -51,16 +51,6 @@ a.tile-title { color: $secondary !important; }
 $card-background-1: $secondary;
 $card-tab-active: $secondary;
 
-.layout-container {
```
Comment (carterqw2) [inline apps/block_scout_web/assets/css/theme/_dai_variables.scss:54]: Is it necessary to remove it from all other themes (here and below)?

**Comment by aaronmgdr** [comment]
right on

File: apps/block_scout_web/assets/css/theme/_dai_variables.scss (hunk 1 above)
Comment (ErinHales) [reply apps/block_scout_web/assets/css/theme/_dai_variables.scss:54]: Yes, you're right. Somebody had used the `!important` specificity on the banner which was one of the things causing the odd styling in dark mode.  So I removed it and nuked all other styling tied to this banner which wasn't the right call.  I should have only removed from the Celo theme.  Thanks for catching this!

**Review by yourbuddyconner — APPROVED** [review]
`!important`!!!

**Pull request merged**